packages/falso/src/lib/permission.ts

```diff
@@ -0,0 +1,45 @@
+import { FakeOptions, fake } from './core/core';
+import { data } from './permission.json';
+
+interface PermissionOptions extends FakeOptions {
+  numeric?: boolean;
+  combination?: boolean;
+}
+
+/**
+ * Generate a random permission.
+ *
+ * @category system
+ *
+ * @example
+ *
+ * randPermission()
+ *
+ * @example
+ *
+ * randPermission({ length: 10 })
+ * randPermission({ numeric: true })
+ * randPermission({ combination: true })
+ * randPermission({ numeric: true, combination: true })
+ *
+ */
+export function randPermission<Options extends PermissionOptions>(
+  options?: Options
+) {
+  if (options?.numeric && options?.combination) {
```

Comment:
I'm not sure I understand this case

Reply:
You can express each combination with a numeric value by adding up the numeric values of individual permissions:
`---` => `0+0+0` => `0`
...
`rwx` => `4+2+1` => `7`

https://www.pluralsight.com/blog/it-ops/linux-file-permissions

Reply:
we can also make it an extra option`numericCombination` 🤔  ...or just get rid of it at all 😅

Reply:
Let's make it simple. We can remove this for now.